content/docs/reference/branding/darkmode-primary-color.md

```diff
@@ -0,0 +1,6 @@
+# Primary Color (Dark Mode)
+- Type: `hex code`
+- Example: `#6F43E7`
+- Default: Pomerium Purple
+
+User's can add a Primary Color that be used in the Theming for the Enterprise Console. This is the color used when users are in Dark Mode.
```

Comment:
```suggestion
Users can add a primary color that will be used for branding and theming for the Pomerium's Enterprise Web Console and information pages. This is the color used when users are in Dark Mode.
```


Though it is set in the console, these colors will be applied to information pages too, right? 

Although, I guess there's really not a dark mode for the user info or error pages....